wagtail_transfer/locators.py

```diff
@@ -127,6 +127,12 @@ def uid_from_json(self, json_uid):
     def find(self, uid):
         # pair up field names with their respective items in the UID tuple, to form a filter dict
         # that we can use for an ORM lookup
+        if type(uid) == tuple:
+            filters = dict(zip(self.fields, uid))
+        elif type(uid) == str:
```

Comment:
Does this issue only occur here: https://github.com/wagtail/wagtail-transfer/blob/main/wagtail_transfer/views.py#L314? In which case, I think it might be better for this method to assume a consistent type to the uid argument, and for the view/some other utility method to handle conversion

Reply:
although that is one place where I found the bug in practice, I'm not sure what the full implications would be for the rest of the codebase in using lookup field values for pages. But I agree if this is the only place that some sort of more testable utility method would be nice, but I'd need to either explore the codebase more or have help speccing that out to be confident in a solution

Reply:
To be honest, it sounds like the key problem that this is solving is UID in query params (there may be other issues, but I think this fix only addresses that?). What happens if you change `request.GET.get` to `request.GET.getlist`? I'm wondering if that may solve the issue too

Reply:
I'll try playing with `request.GET.getlist` more, but at first test it didn't change the error